fummy.py

```diff
@@ -11,11 +11,14 @@
     the main branch
 """
 import os
-import datetime
```

Comment:
This
 ```python  
import datetime 
``` 
causes some fun and games. See [StackOverflow answer] (http://stackoverflow.com/questions/12906402/type-object-datetime-datetime-has-no-attribute-datetime).
 ```python 
from datetime import datetime 
```
The above approach is always recommended